pdf.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
The new page to list PDF files. Uses a custom Vue component `PDFExports` to list URLs and filenames